src/providers/inventory-data.ts

```diff
@@ -36,8 +36,9 @@ export class InventoryData {
       .map(this.extractData);
   }
 
-  return(items: Array<Object>) {
-    return Observable.fromPromise(Promise.resolve());
+  return(tag: string) {
+    return this.authHttp.delete(ApiUrl + Links.rental + '/' + tag)
```

Comment:
Same thing as I commented in another file: this function and the others like it share common functionality and code and should be abstracted away, like you did with `getEndpoint()` elsewhere.